content/examples/getting-started.md

```diff
@@ -1,144 +1,242 @@
-```
-% git clone https://github.com/kr/fs vendor/src/github.com/kr/fs
-Cloning into 'vendor/src/github.com/kr/fs'...
-remote: Counting objects: 18, done.
-remote: Total 18 (delta 0), reused 0 (delta 0), pack-reused 18
-Unpacking objects: 100% (18/18), done.
-Checking connectivity... done.
-% gb build all
-FATAL command "build" failed: failed to resolve package "github.com/pkg/sftp": cannot find package "golang.org/x/crypto/ssh" in any of:
-        /home/dfc/go/src/golang.org/x/crypto/ssh (from $GOROOT)
-        /home/dfc/devel/demo/src/golang.org/x/crypto/ssh (from $GOPATH)
-        /home/dfc/devel/demo/vendor/src/golang.org/x/crypto/ssh
-```
-Nearly there, just missing the `golang.org/x/crypto/ssh` package
-```
-% git clone https://github.com/golang/crypto vendor/src/golang.org/x/crypto
-Cloning into 'vendor/src/golang.org/x/crypto'...
-remote: Counting objects: 1869, done.
-remote: Total 1869 (delta 0), reused 0 (delta 0), pack-reused 1869
-Receiving objects: 100% (1869/1869), 1.19 MiB | 550.00 KiB/s, done.
-Resolving deltas: 100% (1248/1248), done.
-Checking connectivity... done.
-lucky(~/devel/demo) % gb build all
-github.com/kr/fs
-golang.org/x/crypto/ssh
-golang.org/x/crypto/ssh/agent
-github.com/pkg/sftp
-github.com/pkg/sftp/examples/buffered-read-benchmark
-github.com/pkg/sftp/examples/buffered-write-benchmark
-github.com/pkg/sftp/examples/gsftp
-github.com/pkg/sftp/examples/streaming-read-benchmark
-github.com/pkg/sftp/examples/streaming-write-benchmark
-```
-And now it builds. Some things to note
-
-- The package name `all` matches all the packages inside your project's `src/` directory. It's a simple way to build everything, you can use other import paths and globs.
-- There is no way to build your vendored source, it will be built if required to build your code in the `src/` directory.
-
-## More complicated example
-
-For the second example we'll take a project that uses `godep` vendoring and convert it to be a `gb` project. First we'll need to setup a project and get the source
+With the code inside the `gb` project, we can now build the code. We are using the `all` option to have `gb` build all the packages inside the project:
+
+     % gb build all
+     2015/04/29 13:39:44 INFO project root "/home/dfc/devel/sftp"
+     2015/04/29 13:39:44 INFO build duration: 486.967µs map[]
+     2015/04/29 13:39:44 command "build" failed: failed to resolve package "github.com/pkg/sftp": cannot find package "github.com/kr/fs" in any of:
+             /home/dfc/go/src/github.com/kr/fs (from $GOROOT)
+             /home/dfc/devel/sftp/src/github.com/kr/fs (from $GOPATH)
+             /home/dfc/devel/sftp/vendor/src/github.com/kr/fs
+
+When we build the project, it fails because we are missing some dependencies the `gb` tool could not locate. When this happens, we can use the `vendor` command to fetch the dependencies and vendor them inside the project:
+
+     % gb vendor github.com/kr/fs
```

Comment:
i'm sorry but I don't want to have a reference to gb-vendor inside the main documentation. After talking to jason from google and others we decided that gb-vendor, being an obvious wrapper around go get was confusing as gb makes a big point of saying it does _not_ wrap go get. There is a page in plugins/gb-vendor where this example can move to to show an alternative way to initially set up the project